src/lib.rs

```diff
@@ -453,18 +452,19 @@ impl Interpreter {
         self.pc += 2;
     }
 
-    /// Draw the 64x32 bit buffer to a Window.
-    pub fn draw(&mut self, window: &mut Window) {
-        if self.prev_op.is_none() || Self::is_display_op(self.prev_op.unwrap()) {
-            self.graphics.draw(window);
-        }
+    /// Returns the display pixels with a resolution of width by height.
```

Comment:
width by height? These aren't referenced in the function, so we should remove reference to them in the comment.